Modules/_asynciomodule.c

```diff
@@ -119,6 +130,9 @@ class _asyncio.Future "FutureObj *" "&Future_Type"
 static PyObject* future_new_iter(PyObject *);
 static inline int future_call_schedule_callbacks(FutureObj *);
 
+static PyRunningLoopHolder *
```

Comment:
The declaration fits into single line I think. A space between `PyRunningLoopHolder` and `*` is not needed.